recipes-core/packagegroups/packagegroup-ni-desirable.bb

```diff
@@ -25,12 +25,16 @@ RDEPENDS_${PN} += "\
 	cgdb \
 	cifs-utils \
 	elfutils \
+	ffmpeg \
 	file \
 	g++ \
 	gcc \
 	gdb \
 	git \
 	gperf \
+	gstreamer1.0-libav \
+	gstreamer1.0-plugins-good \
+	gstreamer1.0-rtsp-server \
```

Comment:
Did you intend to add x264 too to the list?

Reply:
`x264` is an `ffmpeg` dependency. I didn't add it to the packagegroup directly because it isn't needed as a top-level package.